src/util/color.js

```diff
@@ -1,17 +1,79 @@
 function Color () {}
 
 /**
- * Convert a Scratch color number to a hex string, #RRGGBB.
- * @param {number} color RGB color as a decimal.
+ * Convert a Scratch decimal color to a hex string, #RRGGBB.
+ * @param {number} decimal RGB color as a decimal.
  * @return {string} RGB color as #RRGGBB hex string.
  */
-Color.scratchColorToHex = function (color) {
-    if (color < 0) {
-        color += 0xFFFFFF + 1;
+Color.decimalToHex = function (decimal) {
+    if (decimal < 0) {
+        decimal += 0xFFFFFF + 1;
     }
-    var hex = Number(color).toString(16);
+    var hex = Number(decimal).toString(16);
     hex = '#' + '000000'.substr(0, 6 - hex.length) + hex;
     return hex;
 };
 
+/**
+ * Convert a Scratch decimal color to an RGB color object.
+ * CC-BY-SA
+ * https://stackoverflow.com/questions/29241442/
+ * decimal-to-rgb-in-javascript-and-php
+ * @param {number} decimal RGB color as decimal.
+ * @returns {Object} {r: R, g: G, b: B}, values between 0-255
+ */
+Color.decimalToRgb = function (decimal) {
+    var r = Math.floor(decimal / (256 * 256));
+    var g = Math.floor(decimal / 256) % 256;
+    var b = decimal % 256;
```

Comment:
No reason - because I wasn't thinking too carefully about it and just grabbed a snippet :) I prefer your version for sure. The decimal number shouldn't have more than 24 bits anyway so I think there should never be fractional results.